omc3/correction/handler.py

```diff
@@ -66,6 +66,9 @@ def correct(accel_inst: Accelerator, opt: DotDict) -> None:
     meas_dict = filters.filter_measurement(optics_params, meas_dict, nominal_model, opt)
     meas_dict = model_appenders.add_differences_to_model_to_measurements(nominal_model, meas_dict)
 
+    if opt.arc_by_arc_phase and accel_inst.NAME == 'lhc':
```

Comment:
Do we have to hardcode here?

Reply:
I would go with `Lhc.NAME` but yeah, hardcoded to filter LHC